app/header.jsx

```diff
@@ -32,6 +41,25 @@ class Header extends React.Component {
       </header>
     );
   }
+
+  requestPortscan() {
+    log.debug('Requesting portscan');
+    electron.remote.getCurrentWindow().webContents.send('portscan-requested');
```

Comment:
I would personally put the IPC calls into a middleware. Than you would dispatch an action here, that goes into the middleware. The middleware would in return trigger the call.